TWLight/users/templates/users/filter_section.html

```diff
@@ -17,13 +17,14 @@
       <div class="about-links">
         <a href="{% url 'about' %}" class="twl-links twl-links-underline learn-more-link">
           {% comment %}Translators: On the MyLibrary page (https://wikipedialibrary.wmflabs.org/users/my_library), this appears below introductory text and links to the 'About' page (https://wikipedialibrary.wmflabs.org/about/) {% endcomment %}
-          {% trans "Learn more" %}
-        </a>
+          {% trans "Learn more" %}</a>
```

Comment:
Just a note that I moved these </a>s to prevent the underline from straying into whitespace.